website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -410,11 +410,17 @@ A `default_node_pool` block supports the following:
 
 -> **Note:** This version must be supported by the Kubernetes Cluster - as such the version of Kubernetes used on the Cluster/Control Plane may need to be upgraded first.
 
-* `os_disk_size_gb` - (Optional) The size of the OS Disk which should be used for each agent in the Node Pool. Changing this forces a new resource to be created.
+* `os_disk_size_gb` - (Optional) The size of the OS Disk which should be used for each agent in the Node Pool. 
 
-* `os_disk_type` - (Optional) The type of disk which should be used for the Operating System. Possible values are `Ephemeral` and `Managed`. Defaults to `Managed`. Changing this forces a new resource to be created.
+-> **Note:** Resizing the `default_node_pool` Virtual Machine is done by cycling the system node pool of the cluster. `temporary_name_for_rotation` must be specified when attempting a resize.
+
+* `os_disk_type` - (Optional) The type of disk which should be used for the Operating System. Possible values are `Ephemeral` and `Managed`. Defaults to `Managed`. 
+
+-> **Note:** Changing the `default_node_pool` Virtual Machine disk type is done by cycling the system node pool of the cluster. `temporary_name_for_rotation` must be specified when attempting a resize.
+
+* `os_sku` - (Optional) Specifies the OS SKU used by the agent pool. Possible values include: `Ubuntu`, `CBLMariner`, `Mariner`, `Windows2019`, `Windows2022`. If not specified, the default is `Ubuntu` if OSType=Linux or `Windows2019` if OSType=Windows. And the default Windows OSSKU will be changed to `Windows2022` after Windows2019 is deprecated.
 
-* `os_sku` - (Optional) Specifies the OS SKU used by the agent pool. Possible values include: `Ubuntu`, `CBLMariner`, `Mariner`, `Windows2019`, `Windows2022`. If not specified, the default is `Ubuntu` if OSType=Linux or `Windows2019` if OSType=Windows. And the default Windows OSSKU will be changed to `Windows2022` after Windows2019 is deprecated. Changing this forces a new resource to be created.
+-> **Note:** Changing the `default_node_pool` Virtual Machine disk type is done by cycling the system node pool of the cluster. `temporary_name_for_rotation` must be specified when attempting a resize.
```

Comment:
It bloats the page quite a bit to have the same Note copied under each of the properties - since they're group alphabetically and are listed consecutively can we either consolidate their Note into one i.e.

```suggestion
-> **Note:** Changing `os_sku`, `os_disk_type` or `os_disk_size_gb` of the `default_node_pool` is done by cycling the system node pool of the cluster. `temporary_name_for_rotation` must be specified when attempting a resize.
```

*Alternatively* add the following line behind the description of each property: 
```
`temporary_name_for_rotation` must be specified when changing this.
```

There's validation in the code to throw an error if `temporary_name_for_rotation` hasn't been supplied when changing the `vm_size`, we should also extend it to check that it's been set if one of these properties has been changed.

Reply:
I added both, because I like the one not box, but I also know that people my not mention a box which is not directly below.